src/lib/Liblog/pbs_messages.c

```diff
@@ -405,6 +405,8 @@ char *msg_cannot_set_route_que = "Route queues are incompatible with the partiti
 char *msg_queue_not_in_partition = "Queue %s is not part of partition for node";
 char *msg_partition_not_in_queue = "Partition %s is not part of queue for node";
 char *msg_invalid_partion_in_queue = "Invalid partition in queue";
+char *msg_sched_op_not_permitted = "Operation is not permitted on default scheduler";
+char *msg_sched_part_already_used = "Partition is already associated with other scheduler";
```

Comment:
Getting scheduler name is trivial but getting partition is not straightforward since it logically groups nodes/queues and hence planning to take this as part of subsequent tickets. Also I have added some replies to the same comment in the previous PR. Please look into that.